pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/SchedulerManager.java

```diff
@@ -106,6 +106,7 @@ public SchedulerManager(WorkerConfig workerConfig, PulsarClient pulsarClient, Pu
                                 .blockIfQueueFull(true)
                                 .compressionType(CompressionType.LZ4)
                                 .sendTimeout(0, TimeUnit.MILLISECONDS)
+                                .producerName(config.getWorkerId() + "-scheduler-manager")
```

Comment:
can you also add

.subscriptionRolePrefix(workerConfig.getWorkerId())

Reply:
Producer name needs to be unique, so we better put this in the producer properties instead.

Reply:
Actually, in this case it should be good since there's only 1 writer.